java/ql/src/semmle/code/java/JDK.qll

```diff
@@ -212,14 +212,14 @@ class MethodSystemGetProperty extends Method {
 }
 
 /**
- * An access to a method named `getProperty` on class `java.lang.System`.
+ * Any method access to a method named `getProperty` on class `java.lang.System`.
  */
 class MethodAccessSystemGetProperty extends MethodAccess {
   MethodAccessSystemGetProperty() { getMethod() instanceof MethodSystemGetProperty }
 
   /**
-   * Holds if this call has a compile-time constant first argument with the value `propertyName`.
-   * For example: `System.getProperty("user.dir")`.
+   * Holds true if this is a compile-time constant call for the specified `propertyName`.
+   * Eg. `System.getProperty("user.dir")`.
```

Comment:
Revert this change.
We say "Holds if .." rather than "Holds true if .." (the latter is redundant), and a call cannot be compile-time constant, so the rest of the sentence becomes wrong. We also don't abbreviate in written documentation, so "e.g." is always written "for example".